syntax = "proto3";

import "gene/v1/gene.proto";

package legumeinfo.microservices.genes_service.v1;


service Genes {
  rpc Get (GenesGetRequest) returns (GenesGetReply) {}
}


message GenesGetRequest {
  repeated string names = 1;
}


message GenesGetReply {
  repeated legumeinfo.microservices.gene.v1.Gene genes = 1;
}
